docs/tokenomics/promotional-proof-transactions.md

```diff
@@ -0,0 +1,77 @@
+
+2. **Verification** – A system smart contract validates these proofs through cryptographic verification, ensuring that only legitimate blockchain activity receives rewards.
+
+3. **Karma Allocation** – Upon validation, the user's address receives an initial allocation of Karma, granting them transaction rights and governance power within the network.
+
+4. **Governance Parameters** – The types of proofs accepted, verification methods, and reward values are determined by the network's governance system, allowing the community to adjust the onboarding process as needed.
+
+## Benefits of PPT
+
+- **Zero-Cost Entry** – New users can join the network without already possessing tokens or paying gas fees.
+
+- **Meritocratic Onboarding** – Users with demonstrated blockchain experience receive recognition proportional to their prior contributions.
+
+- **Sybil Resistance** – The cryptographic verification process ensures that only genuine blockchain activity is rewarded, preventing exploitation.
+
+- **Ecosystem Growth** – By reducing barriers to entry, PPT accelerates network adoption and community expansion.
+
+- **Targeted Acquisition** – The network can prioritize specific user behaviors or communities by adjusting which proofs receive higher Karma rewards.
+
+## Use Cases
+
+PPT enables multiple pathways for new users to enter the Status Network ecosystem:
+
+### Cross-Chain Experience Recognition
+
+Users who have built reputation on other chains can transfer some of that standing to Status Network, creating continuity in the web3 experience. For example, a governance participant on Ethereum can receive initial Karma based on their voting history.
+
+### Protocol-Specific Incentives
+
+Status Network can establish targeted onboarding campaigns for users of specific protocols or applications. For instance, users of decentralized social networks might receive PPT rewards to encourage migration to Status Network applications.
+
+### Community Activations
+
+Web3 communities can coordinate collective migrations to Status Network, with PPT ensuring their members maintain appropriate standing and privileges in the new ecosystem.
+
+## Integration with Karma System
+
+PPT works as a foundational element of the broader Karma tokenomics:
+
+- **Initial Allocation** – PPT provides new users with their first Karma, enabling them to perform gasless transactions and participate in governance.
+
+- **Onboarding Tier** – The amount of Karma received through PPT determines a user's initial transaction limits and governance power.
+
+- **Complementary Mechanisms** – Once onboarded, users can continue earning Karma through traditional methods like staking SNT, providing liquidity, and using applications.
+
+## Governance and Evolution
+
+The PPT system is designed to evolve through community governance:
+
+- **Parameter Adjustments** – Karma holders can vote to modify which proofs are accepted and their relative reward values.
```

Comment:
probably Karma + SNT holders

Reply:
the goal is to get people to bridge + stake their SNT (or provide liquidity with it in the native DEX). in both cases they would earn Karma. so i think we can restrict it to Karma holders